From 6b346561de958cd995c920d998bc6429bff39ce8 Mon Sep 17 00:00:00 2001
From: Holger Freyther <zecke@openmoko.org>
Date: Tue, 13 May 2008 18:54:13 +0100
Subject: [PATCH] Disable low level debugging, make the s3c24xx PM debug code depend on DEBUG_LL

Hey,
where are slowly getting into the product state, low level debugging is
supposed to be turned off there. :)

z



From 6448f0a521c05c5ba6672f5908c55583974ce1ed Mon Sep 17 00:00:00 2001
From: Holger Freyther <zecke@openmoko.org>
Date: Fri, 9 May 2008 19:56:31 +0200
Subject: [PATCH] [debug] Disable low level debugging for the product
     Disable low level debugging in the defconfig-gta02 and
     the s3c pm code used printascii which is only available
     with low level debugging, update the Kconfig entry for
     this pm debugging support to depend on low level debugging

Signed-Off-By: Holger Freyther <zecke@openmoko.org>
---
 arch/arm/plat-s3c/Kconfig |    2 +-
 defconfig-gta02           |    5 +----
 2 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/arch/arm/plat-s3c/Kconfig b/arch/arm/plat-s3c/Kconfig
index 31656c3..73bbff3 100644
--- a/arch/arm/plat-s3c/Kconfig
+++ b/arch/arm/plat-s3c/Kconfig
@@ -61,7 +61,7 @@ comment "Power management"
 
 config S3C2410_PM_DEBUG
 	bool "S3C2410 PM Suspend debug"
-	depends on PLAT_S3C && PM
+	depends on PLAT_S3C && PM && DEBUG_LL
 	help
 	  Say Y here if you want verbose debugging from the PM Suspend and
 	  Resume code. See <file:Documentation/arm/Samsung-S3C24XX/Suspend.txt>
diff --git a/defconfig-gta02 b/defconfig-gta02
index c4947b7..13670ee 100644
--- a/defconfig-gta02
+++ b/defconfig-gta02
@@ -159,7 +159,6 @@ CONFIG_CPU_LLSERIAL_S3C2440=y
 #
 # Power management
 #
-CONFIG_S3C2410_PM_DEBUG=y
 # CONFIG_S3C2410_PM_CHECK is not set
 CONFIG_S3C_LOWLEVEL_UART_PORT=2
 
@@ -1693,9 +1692,7 @@ CONFIG_FORCED_INLINING=y
 # CONFIG_SAMPLES is not set
 # CONFIG_DEBUG_USER is not set
 CONFIG_DEBUG_ERRORS=y
-CONFIG_DEBUG_LL=y
-# CONFIG_DEBUG_ICEDCC is not set
-CONFIG_DEBUG_S3C_PORT=y
+# CONFIG_DEBUG_LL is not set
 CONFIG_DEBUG_S3C_UART=2
 
 #
-- 
1.5.6.5

